qiita_db/software.py

```diff
@@ -281,6 +284,17 @@ def create(cls, software, name, description, parameters):
                     for at in atypes]
                 qdb.sql_connection.TRN.add(sql_type, sql_params, many=True)
 
+            # Add the outputs to the command
+            if outputs:
```

Comment:
Just want to confirm that now there are no tests without output. I think is fine due to how it work but want to double check it.